sys/compat/linux/linux_socket.c

```diff
@@ -2111,6 +2116,11 @@ linux_setsockopt(struct thread *td, struct linux_setsockopt_args *args)
 		name = linux_to_bsd_ip_sockopt(args->optname);
 		break;
 	case IPPROTO_IPV6:
+		if (args->optname == LINUX_IPV6_RECVERR &&
+		    linux_ignore_ip_recverr) {
+			/* XXX: as above, for the IPv6 case */
```

Comment:
I would have used the same `XXX` comment `This is a hack to unbreak DNS resolution with glibc 2.30 and above.` to keep it symmetrical.